apps/dash-tsne/assets/styles.css

```diff
@@ -0,0 +1,200 @@
+}
+
+.rc-slider-handle {
+  position: absolute;
+  margin-left: -7px;
+  margin-top: -5px;
+  width: 14px;
+  height: 14px;
+  cursor: pointer;
+  border-radius: 50%;
+  border: solid 2px #3266c1 !important;
+  background-color: #fff;
+}
+.rc-slider-step {
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background: transparent;
+  border-color: #3266c1 !important;
+}
+
+.Select-control {
+  background-color: #f2f3f4;
+  border-radius: 4px;
+  display: table;
+  border-spacing: 0;
+  border-collapse: separate;
+  height: 36px;
+  overflow: hidden;
+}
+
+.js-plotly-plot .plotly .modebar-group {
+  background-color: #f2f3f4 !important;
+}
+
+body {
+  font-size: 1.5em; /* currently ems cause chrome bug misinterpreting rems on body element */
+  line-height: 1.6;
+  font-weight: 400;
+  font-family: "Arimo", "Helvetica Light", "Helvetica Neue", Helvetica,
+    "Helvetica 55 Roman";
+  color: #6c7182;
+}
+
+#learn-more-button > div > div:nth-child(2) > div > p:nth-child(11) > img {
+  width: 40%;
+}
+#graph-3d-plot-tsne > div > div > svg:nth-child(1) {
+  background-color: #f2f3f4 !important;
+}
+
+#graph-3d-plot-tsne > div > div > svg:nth-child(1) > g.bglayer > rect {
+  fill: #f2f3f4 !important;
+}
+
+#graph-bar-nearest-neighbors-word > div > div > svg:nth-child(1) {
+  background-color: #f2f3f4 !important;
+}
+
+#graph-bar-nearest-neighbors-word
+  > div
+  > div
+  > svg:nth-child(1)
+  > g.bglayer
+  > rect {
+  fill: #f2f3f4 !important;
+}
+
+#graph-3d-plot-tsne > div > div > svg:nth-child(1) > g.bglayer > rect {
+  fill: #f2f3f4;
+  fill-opacity: 1;
+  stroke-width: 0;
+}
+
+#react-select-2--value-item {
+  color: #6c7182;
+}
+
+#graph-3d-plot-tsne
+  > div
+  > div
+  > svg:nth-child(3)
+  > g.infolayer
+  > g.legend
+  > rect.bg {
+  stroke: #6c7182 !important;
+  fill: #f2f3f4 !important;
+}
+
+#learn-more-button #button {
+  text-transform: none;
+  font-size: 14px;
+  font-weight: 400;
+  border-radius: 0px;
+  letter-spacing: 0;
+  padding: 0px 15px;
+}
+
+/* ---Different Screen Sizes--- */
+
+@media only screen and (max-width: 768px) {
+  .columns {
+    float: none;
+    width: 90%;
+  }
+
+  .three.columns {
+    margin: auto;
+    width: 75%;
+  }
+
+  .six.columns {
+    margin: 10% 5%;
+    width: auto;
+  }
+
+  .nine.columns {
+    float: left;
+    margin: 0;
+    width: 100%;
+  }
+
+  background {
+    width: 100%;
+  }
+
+  #app-title {
+    text-align: center !important;
+  }
+
+  .header_img {
+    height: 40px;
+    float: left;
+    margin: 0 !important;
+  }
+
+  #learn-more-button {
+    text-align: center;
+    padding-right: 0 !important;
+  }
+
+  #learn-more-button > div > div:nth-child(1) {
+    padding-right: 7% !important;
+  }
+
+  #learn-more-button > div > div:nth-child(2) {
+    padding-right: 7%;
+  }
+
+  #learn-more-button > div > div:nth-child(3) {
+    padding-right: 15%;
+  }
+}
+@media only screen and (max-width: 550px) {
+  #plotly-image {
+    height: 35px !important;
```

Comment:
It's generally not advisable to put `!important` in css unless necessary. I think for some examples, the css call is so specific that `!important` isn't needed.